learn/index.ftd

```diff
@@ -7,6 +7,7 @@
 
 
 -- lib.page: Admint
+active-page: learn/
```

Comment:
@priyanka9634 try not to add any such new property in `page` component.

Reply:
To achieve this fixes required in sitemap and toc processor.